examples/jsm/loaders/USDZLoader.js

```diff
@@ -254,7 +254,7 @@ class USDZLoader extends Loader {
 
 			if ( id !== undefined ) {
 
-				const def = `def "%{id}"`;
+				const def = `def "${id}"`;
```

Comment:
Was this a bug? The `id` replacement only works when `$` is used.

Reply:
Probably.